### ISSUE TITLE
MongoDB Error while running tests

### ISSUE THREAD

**Issue by steveg152** [FrontendMasters/api-design-node-v3#24]
When running npm run test-models during lesson-3, there are issues with the MongoDB connection.  There was nothing going over any configuration for MongoDB for this exercise.  Even the solution branch fails the tests because of the mongo connection.

    MongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]

      at Pool.<anonymous> (node_modules/mongodb-core/lib/topologies/server.js:564:11)
      at Connection.<anonymous> (node_modules/mongodb-core/lib/connection/pool.js:317:12)
      at Socket.<anonymous> (node_modules/mongodb-core/lib/connection/connection.js:246:50)

**Comment by calffey** [comment]
I was running into the same issue.

You have to have MongoDB installed on your computer and have it running. You can find instructions on how to do that here -> https://github.com/mongodb/homebrew-brew

**Comment by Mfhodges** [comment]
I also ran into this issue. 
@calffey's resource helped me a lot. What I had to do specifically was open a new terminal and then run `$ brew services start mongodb-community`. Check `usr/local/` ( or `usr/local/Cellar` for OS) to see what mongodb-community you have. 

If you cannot run `$ mongo` in a new terminal then you will need to add `export PATH="$PATH:/usr/local/Cellar/mongodb-community@3.6/3.6.17/bin"` to your .bash_profile ( again swapping the version for whatever you actually have.
